google/cloud/pubsub/samples/samples.cc

```diff
@@ -341,18 +341,18 @@ void UpdateDeadLetterSubscription(
 
 void ReceiveDeadLetterDeliveryAttempt(
     google::cloud::pubsub::Subscriber subscriber,
-    google::cloud::pubsub::Subscription const& subscription,
+    google::cloud::pubsub::Subscription const&,
```

Comment:
Can this now-unused parameter be removed from here (and also `Subscribe()`, `SubscribeErrorListener()`, and `SubscribeCustomAttributes()`) along with the suggested change to `examples::SubscriberCommand`?

Reply:
Good catch, I though about it as I was making these changes and then promptly forgot.